content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
-
-            <div class="description">
-
-              <div class="ui list inverted relaxed divided">
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Automatic building</div>
-                    <div class="description">on every commit</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Multi-version handling</div>
-                    <div class="description">from branches or tags</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Private versions</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Live preview</div>
-                    <div class="description">of all pull requests</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Unlimited projects & users</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">API access</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Enhanced full-text search</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Multiple formats</div>
-                    <div class="description">HTML, PDF, and EPUB</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Email build failure notifications</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">2 day support response time</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">2 concurrent builds</div>
-                  </div>
-                </div>
-              </div>
+  {% block pricing_community_intro %}
+    <div class="ui padded container">
+      <div class="ui clearing inverted teal segment">
+        <div class="ui stackable grid">
 
-            </div>
+          <div class="sixteen wide tablet twelve wide computer middle aligned column">
+            <i class="fad fa-trees icon"></i>
+            Host <b>open source and community projects</b> for free with Read the Docs Community.
```

Comment:
I do wonder about putting this callout lower on the page. Perhaps above the feature checklists?

Reply:
Hrm, good question, both look good. Here's above the feature list:

![image](https://user-images.githubusercontent.com/1140183/201786950-69430054-aaa5-4281-bd57-584d0c66b3e9.png)

Reply:
I have slight preference to having it first, but dropped it down for now. I'll note on #167 but I think it does look good between plans/features, so not a big issue at all either way.

Reply:
Yea, it looks a bit redundant with that placement right above the feature list 🤔

Reply:
Yeah, it's an odd piece. I'm happy with it as it is here and #167 can revisit some fixes perhaps.